docs/user-guide/creating-application/gitops-config.md

```diff
@@ -0,0 +1,89 @@
+# GitOps Configuration
+
+{% hint style="warning" %}
+The 'GitOps Configuration' page appears only if the super-admin has enabled 'Allow changing git repository for application' in [Global Configurations → GitOps](../global-configurations/gitops.md).
+
+### Feature Flag
+
+* **For disabling**: `FEATURE_USER_DEFINED_GITOPS_REPO_ENABLE=false`
```

Comment:
We also need to tell that this feature flag needs to go in ConfigMap of dashboard of the devtron instance. Maybe Attach an image of resource browser?

Reply:
Done. Added steps and a gif in GC's GitOps doc and removed from here.